tsconfig.publish.json

```
@@ -0,0 +1,11 @@
+{
+  "extends": "./tsconfig.json",
+  "compilerOptions": {
+    "noEmit": false,
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    "noEmitOnError": true,
+    "outDir": "dist"
+  },
+  "include": ["lib/**/*"]
```

need to update this to include the root index.js

```suggestion
  "include": ["lib/**/*", "index.js"]
```